frame/support/src/lib.rs

```diff
@@ -127,80 +129,48 @@ impl TypeId for PalletId {
 	const TYPE_ID: [u8; 4] = *b"modl";
 }
 
-/// Generate a new type alias for [`storage::types::StorageValue`],
-/// [`storage::types::StorageMap`], [`storage::types::StorageDoubleMap`]
-/// and [`storage::types::StorageNMap`].
+/// Generate a [`#[pallet::storage]`](pallet_macros::storage) alias outside of a pallet.
 ///
-/// Useful for creating a *storage-like* struct for test and migrations.
+/// This storage alias works similar to the [`#[pallet::storage]`](pallet_macros::storage)
+/// attribute macro. It supports [`storage::types::StorageValue`],
+/// [`storage::types::StorageMap`], [`storage::types::StorageDoubleMap`] and
+/// [`storage::types::StorageNMap`]. The main difference to the normal
```

Comment:
This gets a bit verbose in the generated docs, I would recommend changing each link in this little section to be like e.g.:

```rust
/// [`StorageMap`](`storage::types::StorageMap`)
```

This way the reader just sees `StorageMap` but can easily figure out where that type is located by clicking it.